tests/src/compare_delphes_converter_outputs.cpp

```diff
@@ -98,14 +98,26 @@ template <typename DelphesT> std::vector<GenParticle*> getAssociatedMCParticles(
  */
 template <typename DelphesT>
 bool compareMCRelations(const DelphesT* delphesCand, edm4hep::ReconstructedParticle edm4hepCand,
-                        const edm4hep::MCRecoParticleAssociationCollection& associations) {
-  const auto delphesGenParticles = getAssociatedMCParticles(delphesCand);
-  const auto edm4hepMCParticles  = getAssociatedMCParticles(edm4hepCand, associations);
+                        const edm4hep::RecoMCParticleLinkCollection& associations) {
+  auto delphesGenParticles = getAssociatedMCParticles(delphesCand);
+  auto edm4hepMCParticles  = getAssociatedMCParticles(edm4hepCand, associations);
 
   if (delphesGenParticles.size() != edm4hepMCParticles.size()) {
     return false;
   }
 
+  // In some cases the order of the vectors is not exactly the same, because
+  // Delphes(!) seems to write them in a different order from time to time even
```

Comment:
Does that mean delphes uses pointers (or similar) for map keys and that is affected by memory randomization?